source/vibe/inet/webform.d

```diff
@@ -259,7 +259,20 @@ private bool parseMultipartFormPart(InputStream stream, ref FormFields form, ref
 		auto data = cast(string)stream.readUntil(boundary);
 		form.addField(name, data);
 	}
-	return stream.readLine(max_line_length) != "--";
+	
+	enforce(stream.leastSize >= 2);
+	auto ub = stream.peek();
```

Comment:
For me the peek returns `--[boundary]--`, so maybe I should read the boundary + 2